src/amber.cr

```diff
@@ -49,10 +55,17 @@ module Amber
       @host = "0.0.0.0"
       @port_reuse = true
       @key_generator = Amber::Support::CachingKeyGenerator.new(
-        Amber::Support::KeyGenerator.new(secret, 1000)
+        Amber::Support::KeyGenerator.new(secret, 5)
       )
       @pubsub_adapter = WebSockets::Adapters::MemoryAdapter
       @redis_url = "redis://localhost:6379"
+      @session = {
+        :key       => "session_id",
+        :store     => :cookie,
+        :expires   => 2 * 60 * 60, # 2 Hours
```

Comment:
Session should last as long as the browser is open like the other MVC's instead of 2 hours.